application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -93,9 +93,38 @@
         #if($isChecked)X#end
       #else
         &lt;label&gt;
-          &lt;input type="#if($isRadio)radio#{else}checkbox#end"
-            #if($isCurrentUser)id="xpoll-proposal$proposal.hashCode()"#end name="$user" value="$proposal" $checked
-            $disabled /&gt;
+         ## condorcet method
+         #if ($pollObj.getValue('type') == '2')
+           #set ($indexOfProposal = $votes.indexOf($proposal) + 1)
+           #set ($noOption = '')
+           #if ($indexOfProposal &lt;  1)
+             #set ($noOption = 'selected')
```

Comment:
You could simply call this variable `selected`, right?

Reply:
Yes. but it think it might be a bit ambiguous. With that variable i want to set the first, disabled option as selected or not. If the variable name is "selected" and the value is selected as well, one would think that the proposal is..selected. But that's not the case, because we are selecting the option that determines the proposal as NOT selected. Ok. This sounds like a mouthful. 

Let me know what you think.